packages/quantic/force-app/main/default/lwc/quanticResultChildren/quanticResultChildren.js

```diff
@@ -173,7 +173,10 @@ export default class QuanticResultChildren extends LightningElement {
   }
 
   get shouldDisplayNoChildrenMessage() {
```

Comment:
I'm pretty sure we can simplify the condition to this:
```
  get shouldDisplayNoChildrenMessage() {
    if (!this.isFirstLevelChildCollection || !this.areAllChildResultsLoaded) {
      return false;
    }
    return !this.moreResultsFound;
  }
```
In other words, we display the `NoChildrenMessage` only in one case, which is when the end user has clicked the `load all items` but no more results have been found.

Reply:
nice! you're right, will refactor

Reply:
ohhh yea ok I was about to write that the two were not equivalent at all, but you're right, it's only whenever we tried to load children and there was none to load